gui/js/updater.window.js

```diff
@@ -35,9 +48,9 @@ module.exports = class UpdaterWM extends WindowManager {
     autoUpdater.autoDownload = false
     autoUpdater.on('update-available', info => {
       this.clearTimeoutIfAny()
-      log.info({ update: info, skipped: this.skipped }, 'Update available')
       // Make sure UI don't show up in front of onboarding after timeout
       if (!this.skipped) {
+        log.info({ update: info, skipped: this.skipped }, 'Update available')
```

Comment:
Is there a point in logging the `skipped` value now? And wasn't the point of this log line to know when we were skipping updates that were available?

Maybe we should still log something when we have an available update but skip it.